app/controllers/assets.php

```diff
@@ -39,4 +39,49 @@ function list_json() {
 
         echo json_encode($assets_list);
     }
+
+    function update() {
+        $is_success = null;
+
+        $id = null;
+        $name = null;
+        $description = null;
+
+        if (!empty($_GET['id'])) {
+            $id = $_REQUEST['id'];
+        }
+
+        if ( null == $id ) {
+            //TODO: Set warning or maybe ignore
+            header("Location: view");
+        }
+
+        if (isset($_POST['submit'])) {
+            $id = $_POST['id'];
+            $is_success = $this->assets_model->update(
+                $_POST['id'], // asset id
```

Comment:
It is. It'll through a warning if not present. But we have been conveniently ignoring warnings and assuming some scenarios.. :P

Anyway, as far as we use prepared statements, it can't mess with DB with injection and stuff. But yes, it is still a problem.